File: demo/src/main/java/com/example/demo/entity/Student.java
package com.example.demo.entity;

import com.fasterxml.jackson.annotation.JsonProperty;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
@Builder
@AllArgsConstructor
public class Student {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @JsonProperty("id")
    private Long id;
    @JsonProperty("name")
    private String name;
    @JsonProperty("roll_no")
    private Long rollNo;
    @JsonProperty("address")
    private String address;

}

File: student-angular-app/src/app/MyComponents/model/student.ts
export class Student {

  private _name!: string;
  private _id!: number;
  private _rollNo!: number;
  private _address!: string;

  public get address(): string {
    return this._address;
  }
  public set address(value: string) {
    this._address = value;
  }
  public get rollNo(): number {
    return this._rollNo;
  }
  public set rollNo(value: number) {
    this._rollNo = value;
  }

  public get id(): number {
    return this._id;
  }
  public set id(value: number) {
    this._id = value;
  }

  public get name(): string {
    return this._name;
  }
  public set name(value: string) {
    this._name = value;
  }
}
